packages/help-text/package.json

```diff
@@ -83,7 +83,7 @@
         "@spectrum-web-components/shared": "1.4.0"
     },
     "devDependencies": {
-        "@spectrum-css/helptext": "6.0.1"
+        "@spectrum-css/helptext": "7.1.0"
```

Comment:
This package is already listed in the changesets file `.changeset/brave-crews-win.md`